CHANGELOG.md

```diff
@@ -0,0 +1,331 @@
+`--nginx` on the command line. This plugin is still in its early stages 
+so we recommend you use it with some caution and make sure you have a 
+backup of your Nginx configuration. 
+* Support for the `DNS` challenge in the `acme` library as well as `DNS` 
+support in Certbot's `manual` plugin. This allows you to create DNS 
+records to prove to Let's Encrypt you control the requested the domain 
+name. To use this feature, include `--manual --preferred-challenges dns` 
+on the command line. 
+* Help with enabling Extra Packages for Enterprise Linux (EPEL) on 
+CentOS 6 when using `certbot-auto`. To use `certbot-auto` on CentOS 6, 
+the EPEL repository has to be enabled. `certbot-auto` will now prompt 
+users asking them if they would like the script to enable this for them 
+automatically. This is done without prompting users when using 
+`letsencrypt-auto` or if `-n/--non-interactive/--noninteractive` is 
+included on the command line. 
+
+More details about these changes can be found on our GitHub repo:
+https://github.com/certbot/certbot/issues?q=is%3Aissue+milestone%3A0.9.0+is%3Aclosed 
+
+# 0.8.1
+## 06/14/2016
+
+
+* Preserving a certificate's common name when using `renew` 
+* Save webroot values for renewal when they are entered interactively 
+* Problems with an invalid user-agent string on OS X 
+* Gracefully reporting the Apache plugin isn't usable when Augeas is not installed 
+* Experimental support for Mageia has been added to `certbot-auto`
+
+More details about these changes can be found on our GitHub repo:
+https://github.com/certbot/certbot/issues?q=is%3Aissue+milestone%3A0.8.1+
+
+# 0.8.0
+## 06/02/2016
+
+* The main new feature in this release is the `register` subcommand which 
+can be used to register an account with the Let's Encrypt CA. 
+* Additionally, you can run `certbot register --update-registration` to 
+change the e-mail address associated with your registration. 
+
+More details about these changes can be found on our GitHub repo:
+https://github.com/certbot/certbot/issues?q=is%3Aissue+milestone%3A0.8.0+
+
+# 0.7.0
+## 05/27/2016
+
+* `--must-staple` to request certificates from Let's Encrypt with the 
+OCSP must staple extension 
+* automatic configuration of OSCP stapling for Apache 
+* requesting certificates for domains found in the common name of a 
+custom CSR 
+* a number of bug fixes 
+
+More details about these changes can be found on our GitHub repo:
+https://github.com/certbot/certbot/issues?q=milestone%3A0.7.0+is%3Aissue 
+
+# 0.6.0
+## 05/12/2016 
+
+* Renamed the client from `letsencrypt` to `certbot`
+* Fixed a small json deserialization error
+* Versioned the datetime dependency in setup.py
+* Preserve domain order in generated CSRs
+* Some minor bug fixes
+
+More details about these changes can be found on our GitHub repo:
+https://github.com/certbot/certbot/issues?utf8=%E2%9C%93&q=is%3Aissue%20milestone%3A0.6.0%20is%3Aclosed%20
```

Comment:
nit: In the other links Peter or I removed `utf8=%E2%9C%93` from the query string. Let GH determine whether or not serve content as UTF-8.